certified-connectors/bitskout/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+	"properties": {
+		"connectionParameters": {
+			"api_key": {
+				"type": "securestring",
+				"uiDefinition": {
+					"displayName": "API Key",
+					"description": "The API Key for this api",
+					"tooltip": "Provide your API Key",
+					"constraints": {
+						"tabIndex": 2,
+						"clearText": false,
+						"required": "true"
+					}
+				}
+			}
+		},
+		"iconBrandColor": "#007ee5",
```

Comment:
@saferoombiz the icon must contain a non-transparent, non-white color (#ffffff) background and not-default color (#007ee5) that matches your specified icon background color. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector)
Please change this to your icon.png background color.